From fc4cc7c0ac671ada8ef365795cdd89c86af09b9d Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Fri, 31 Jul 2020 11:21:09 +0700
Subject: [PATCH 419/504] ARM: dts: r8a7745: Add tddmac support to SoC dtsi

This patch add Image Extraction Direct Memory Access Controller (tddmac)
module to RZ/G1M device tree.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7745.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7745.dtsi b/arch/arm/boot/dts/r8a7745.dtsi
index 4aebd97..c454e76 100644
--- a/arch/arm/boot/dts/r8a7745.dtsi
+++ b/arch/arm/boot/dts/r8a7745.dtsi
@@ -1370,6 +1370,14 @@
 			status = "disabled";
 		};
 
+		tddmac@0xfea00000 {
+			compatible = "renesas,vspm-tddmac","renesas-vspm";
+			reg = <0 0xfea00000 0 0x0200>;
+			interrupts = <0 285 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7745_CLK_2DDMAC>;
+			power-domains = <&cpg_clocks>;
+		};
+
 		vsp@fe928000 {
 			compatible = "renesas,vsp1";
 			reg = <0 0xfe928000 0 0x8000>;
-- 
2.7.4

